hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -163,7 +165,10 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
 
       if (base != null) {
         metaClients.run(client -> {
-          client.alter_table(database, tableName, tbl);
+          client.alter_table(database, tableName, tbl,
```

Comment:
Nit: Iceberg uses 4 spaces for continued indentation and `ImmutableMap` from Guava.

What about this?

```
EnvironmentContext envContext = new EnvironmentContext(
    ImmutableMap.of(StatsSetupConst.DO_NOT_UPDATE_STATS, StatsSetupConst.TRUE)
);
client.alter_table(database, tableName, tbl, envContext);

```

Reply:
Sure thing, sorry